src/object/srv_obj.c

```diff
@@ -1186,8 +1186,8 @@ obj_fetch_shadow(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch,
 	}
 
 	obj_iod_idx_vos2parity(iod_nr, iods);
-	rc = vos_fetch_begin(coh, oid, epoch, cond_flags, dkey, iod_nr, iods,
-			     VOS_FETCH_RECX_LIST, NULL, &ioh, NULL);
+	rc = vos_fetch_begin(coh, oid, epoch, dkey, iod_nr, iods,
+                cond_flags | VOS_FETCH_RECX_LIST, NULL, &ioh, NULL);
```

Comment:
(style)  code indent should use tabs where possible

Reply:
If you use vim, I have a vim style file that follows the rules for you.  In this case, it would align 1190 with 1189 with tabs at the start and only spaces to make up the difference.